### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by Mat-Moo** [forkineye/ESPixelStick#571] [General]
638 commits since last v4 release, any chance of a new release :)

**Comment by MartinMueller2003** [comment]
You can get the latest version pre-built here:
https://drive.google.com/file/d/1DG3wz8lbYNc371OIbeOqkzMaY5Zd2SKA/view?usp=sharing

**Comment by cybercop23** [comment]
@MartinMueller2003 do you have one for the newest one? Looks like they are all expired now.

**Comment by MartinMueller2003** [comment]
The google drive link never expires. 
You can also get the latest here: https://github.com/MartinMueller2003/ESPixelStick/releases
Or use the new web installer: https://espixelstickwebflasher.from-ct.com:5000

**Comment by MartinMueller2003** [comment]
Google drive links are no longer being updated. Latest versions are found on the web flash tool.
